### PR TITLE
Add activate handler.

### PR DESCRIPTION
**Pull request by mikecdavis**
## Summary
* Add activate handler

To keep the scope small I'm breaking out the changes in PR #133 into several smaller PRs.

### CODE REVIEW

File: pkg/handlers/activate.go [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+			return
+		}
+
+		d, err := optlyClient.ActivateExperiment(&e, uc, disableTracking)
+		if err != nil {
+			RenderError(err, http.StatusInternalServerError, w, r)
+			return
+		}
+
+		// TODO implement enabled filtering
```
Comment (mjc1283) [inline pkg/handlers/activate.go]: JIRA?

File: pkg/handlers/activate.go [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+	for _, key := range query["experimentKey"] {
+		experimentSet[key] = struct{}{}
+	}
+
+	// Add explicit features
+	for _, key := range query["featureKey"] {
+		featureSet[key] = struct{}{}
+	}
+
+	logger.Debug().Msg("iterate over all experiment decisions")
```
Comment (mjc1283) [inline pkg/handlers/activate.go]: This doesn't seem very useful. Maybe better to include the keys that will be decided on?

File: pkg/handlers/activate.go [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+		}
+
+		// TODO implement enabled filtering
+		decisions = append(decisions, d)
+	}
+
+	render.JSON(w, r, decisions)
+}
+
+func getUserContext(r *http.Request) (entities.UserContext, error) {
```
Comment (mjc1283) [inline pkg/handlers/activate.go]: Maybe this function should be reused in the track or override handlers.

File: pkg/handlers/activate.go (hunk 1 above)
Comment (mikecdavis) [reply pkg/handlers/activate.go]: I adding the functionality.

File: pkg/handlers/activate.go (hunk 2 above)
Comment (mikecdavis) [reply pkg/handlers/activate.go]: moved to a debug inside the loop

**Pull request merged**